src/webui/api/synccontroller.cpp

```diff
@@ -550,7 +550,10 @@ void SyncController::torrentPeersAction()
             peer[KEY_PEER_COUNTRY] = Net::GeoIPManager::CountryName(pi.country());
         }
 
-        peers[pi.address().ip.toString() + ':' + QString::number(pi.address().port)] = peer;
+        if (!pi.address().ip.toString().contains('.'))  // IPv6
```

Comment:
Maybe I'm misunderstanding, but how can an IPv4 not contain a `.` character?

Reply:
It just say that IP strings don't contain a `.` character are IPv6 addresses.

Reply:
Sorry, brain fart.